pkgs/development/python-modules/pytest-astropy-header/default.nix

```diff
@@ -37,12 +33,7 @@ buildPythonPackage rec {
 
   checkInputs = [
     pytestCheckHook
-    pytest-cov
     numpy
-    scipy
-    h5py
-    scikitimage
-    astropy
```

Comment:
These can be removed from the arguments now.

Reply:
You're right. Fixed with bf012b5.